integration-test/src/main/java/com/sequenceiq/it/cloudbreak/testcase/e2e/l0promotion/InternalSdxSshAndCmAccessTest.java

```diff
@@ -0,0 +1,144 @@
+
+    @Test(dataProvider = TEST_CONTEXT)
+    @Description(
+            given = "there is a running Manowar SDX cluster in available state",
+            when = "SSH to the SDX nodes with 'admin' then acting user with new workload password, where the 'post-install' recipe had been installed.",
+            and = "log in to the Cloudera Manager, where 'knox-IDBROKER-BASE' role config mappings then related Cloudera Manager user details have been read",
+            then = "SSH and Cloudera Manager API access should be successful, 'post-install' recipe files and requested Cloudera Manager details should be" +
+                    " present"
+    )
+    public void testSdxSshAndCmAccessWithNewWorkloadPassword(TestContext testContext) throws IOException {
+        String freeIpa = testContext.given(FreeIpaTestDto.class).when(freeIpaTestClient.describe()).getResponse().getName();
+        String sdxInternal = resourcePropertyProvider().getName();
+        String cluster = resourcePropertyProvider().getName();
+        String clouderaManager = resourcePropertyProvider().getName();
+        String recipeName = resourcePropertyProvider().getName();
+        String stack = resourcePropertyProvider().getName();
+        String filePath = "/post-install";
+        String fileName = "post-install";
+        String masterInstanceGroup = "master";
+        String idbrokerInstanceGroup = "idbroker";
+        String newWorkloadPassword = "Admin123@";
+        String workloadUsername = getWorkloadUsername(testContext);
+
+        testContext
+                .given(clouderaManager, ClouderaManagerTestDto.class)
+                .given(cluster, ClusterTestDto.class).withBlueprintName(getDefaultSDXBlueprintName()).withValidateBlueprint(Boolean.FALSE)
+                .withClouderaManager(clouderaManager)
+                .given(RecipeTestDto.class).withName(recipeName).withContent(generateRecipeContent())
+                .withRecipeType(POST_CLOUDERA_MANAGER_START)
+                .when(recipeTestClient.createV4())
+                .given(masterInstanceGroup, InstanceGroupTestDto.class).withHostGroup(MASTER).withNodeCount(1).withRecipes(recipeName)
+                .given(idbrokerInstanceGroup, InstanceGroupTestDto.class).withHostGroup(IDBROKER).withNodeCount(1).withRecipes(recipeName)
+                .given(stack, StackTestDto.class).withCluster(cluster).withInstanceGroups(masterInstanceGroup, idbrokerInstanceGroup)
+                .given(sdxInternal, SdxInternalTestDto.class)
+                .withCloudStorage(getCloudStorageRequest(testContext))
+                .withStackRequest(key(cluster), key(stack))
+                .when(sdxTestClient.createInternal(), key(sdxInternal))
+                .await(SdxClusterStatusResponse.RUNNING)
+                .awaitForInstance(getSdxInstancesHealthyState())
+                .then((tc, testDto, client) -> sshJUtil.checkFilesOnHostByNameAndPath(testDto, client, List.of(MASTER.getName(), IDBROKER.getName()),
```

Comment:
I would check for a log file or something like that, for this I would not create a recipe